app/controllers/photo-albums/photo-album/photos/show.js

```diff
@@ -4,7 +4,9 @@ import Controller from '@ember/controller';
 export default Controller.extend({
   session: service(),
   advanceToPhoto(delta) {
-    const photos = this.model.photoAlbum.get('photos');
+    const photos = this.model.photoAlbum
+      .get('photos')
+      .sortBy('exifDateTimeOriginal', 'createdAt');
```

Comment:
can't this sorting take place in the `PhotoAlbum` model? For clarification, see `models/form/form.js`, specifically `get sortedQuestions()`.